### PR TITLE
GitHub CI actions

### PR DESCRIPTION
**Pull request by ChGen**
GitHub CI actions + `.gitattributes`

### CODE REVIEW

**Comment by ChGen** [comment]
@dgarus , please review.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      - '*'
+jobs:
+  build:
+    name: Build release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.3
+      - uses: actions/setup-java@v2
+        with:
+          java-version: 8
```
Comment (dgarus) [inline .github/workflows/release.yml:14]: why 8?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+name: Create Maven release
+on:
+  push:
+    tags:
+      - '*'
```
Comment (dgarus) [inline .github/workflows/release.yml:5]: Does `*` mean for any tag?

**Review by dgarus — CHANGES_REQUESTED** [review]

**Comment by ChGen** [comment]
@dgarus , please review.

**Review by dgarus — APPROVED** [review]

**Comment by ChGen** [comment]
@olenagerasimova , please review.

**Review by olenagerasimova — APPROVED** [review]
thanks

**Pull request merged**